_rules/form-field-label-descriptive-cc0f0a.md

```diff
@@ -218,7 +218,7 @@ Both the `div` and the `span` elements are [programmatic labels][programmatic la
 
 ```html
 <html lang="en">
-	<div id="shipping">Shipping address</div>
+	<div id="shipping">Shipping</div>
 	<span id="name" style="display: none">Name</span>
 	<input id="shipping-name" type="text" name="name" aria-labelledby="shipping name" />
 </html>
```

Comment:
Maybe the way to make this example better is by adding another field to create the confusion:
```html
<html lang="en">
	<div id="shipping">Shipping</div>
	<span id="name" style="display: none">Name</span>
	<input id="shipping-name" type="text" name="name" aria-labelledby="shipping name" />
	<span id="address" style="display: none">Address</span>
	<input id="shipping-address" type="text" name="address" aria-labelledby="shipping address" />
</html>
```
Now, it is clear that both input have the same visible context ("Shipping") which is not descriptive enough, even though the programmatic context (accessible names) are good enough.